JYP-iOS/JYP-iOS/Sources/Presenter/Planner/PlannerRoute/PlannerRouteViewController.swift

```diff
@@ -108,17 +118,46 @@ class PlannerRouteViewController: NavigationBarViewController, View {
             .bind(to: reactor.action)
             .disposed(by: disposeBag)
         
+        routeCollectionView.rx.itemSelected
+            .map { .tapRouteCell($0) }
+            .bind(to: reactor.action)
+            .disposed(by: disposeBag)
+        
         pikmiRouteCollectionView.rx.itemSelected
             .map { .tapPikmiRouteCell($0) }
             .bind(to: reactor.action)
             .disposed(by: disposeBag)
         
+        reactor.state
+            .map(\.order)
+            .withUnretained(self)
+            .bind { this, order in
+                this.setNavigationBarTitleText("DAY \(order + 1)")
+            }
+            .disposed(by: disposeBag)
+        
+        reactor.state
+            .map(\.date)
+            .withUnretained(self)
+            .bind { this, date in
+                this.setNavigationBarSubTitleText(DateManager.dateToString(format: "M월 d일", date: date))
+            }
+            .disposed(by: disposeBag)
+        
         reactor.state
             .map(\.routeSections)
             .withUnretained(self)
             .bind { this, sections in
                 let layout = this.makeRouteLayout(sections: sections)
                 this.routeCollectionView.collectionViewLayout = layout
+                
+                if let section = sections.first {
+                    if section.items.isEmpty {
+                        this.emptyLabel.isHidden = false
+                    } else {
+                        this.emptyLabel.isHidden = true
+                    }
```

Comment:
```suggestion
                        this.emptyLabel.isHidden = !section.items.isEmpty
```
이런 식으로 줄일 수 있지 않을까요?

Reply:
인정,, 좋은 피드백 감사합니다 .. !